index.js

```diff
@@ -726,7 +726,11 @@ function nested (laterCode, name, key, schema, externalSchema, fullSchema, subKe
         code += `
           else json+= null
         `
-      } else throw new Error(`${schema} unsupported`)
+      } else {
```

Comment:
Probably just to be safe we can add a check here to verify that you are passing an empty schema.